spec/games_spec.rb

```diff
@@ -61,4 +61,29 @@
       expect(game_report).to eq(expected_final_report)
     end
   end
+
+  describe '#death_report' do
+    subject(:games){described_class.new('spec/fixtures/example.log')}
+
+    it 'returns an array' do
+      game_report = subject.death_report
```

Comment:
"game_report" doesn't reflect correctly what it is here, use death_report instead.
If you change your subject, you can skip this line and work directly with the `expect(death_report).to <rest of code>`